### ISSUE TITLE
direction of bank transfer: differentiate between payin and payout

### ISSUE THREAD

**Issue by schmittlauch** [liberapay/liberapay.com#866]
I just saw that on the payin (add money to your wallet) page the wording may be a bit confusing: The msgid of the description for bank transfer is "To an account in Europe" there, although for pay "*From* an account in Europe" is more appropriate.
I'm not fixing that myself so far because I suspect you use https://github.com/liberapay/liberapay.com/blob/6dce66fa4c38daf9a7ddfc244e9dba754125429e/www/%25username/wallet/payin/%25back_to.spt both for payin and payout, and I'm not familiar with the project structure so far.

**Comment by Changaco** [comment]
No, "To an account in Europe" is correct. The money can be sent from pretty much any country in the world, but the destination will be France or Luxembourg.

**Comment by schmittlauch** [comment]
Oh, then it was a misunderstanding on my side. I don't know whether it was just me or whether this could use some clarification?
